pkg/sqlite/load.go

```diff
@@ -104,8 +110,85 @@ func (s *sqlLoader) addOperatorBundle(tx *sql.Tx, bundle *registry.Bundle) error
 	if err != nil {
 		return err
 	}
+	substitutesFor, err := bundle.SubstitutesFor()
+	if err != nil {
+		return err
+	}
+
+	// Update the replaces that reference the substituted-for bundle
+	if substitutesFor != "" {
```

Comment:
I think the substitutes-for processing generally needs to happen after all bundles have been added, right?

If I add a bundle after this `add` has run that would have been substituted by this bundle, it will be missed. There's an existing division in the codebase around this: 1. load all bundles, 2. wire them up.

Reply:
I can add a test for that but we should be ok in that respect because the graph gets rebuilt everytime from information stored during 1.